tests/test_prices.py

```diff
@@ -178,6 +178,25 @@ def test_set_instance_values():
     assert instance.price_gross == Money(30, 'BTC')
 
 
+def test_init_taxedmoney_model_field():
+    instance = Model(price=TaxedMoney(Money(25, 'BTC'), Money(30, 'BTC')))
+    assert instance.price_net == Money(25, 'BTC')
+    assert instance.price_gross == Money(30, 'BTC')
+
+
+def test_init_taxedmoney_model_field_validation():
+    with pytest.raises(ValueError):
+        instance = Model(price=TaxedMoney(Money(25, 'USD'), Money(30, 'USD')))
+        instance.full_clean()
```

Comment:
Which line raises the exception? Please limit the exception catching scope to a single expression.